systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -186,6 +213,16 @@ void testMirrorMaker2() {
             internalKafkaClient.receiveMessagesPlain()
         );
 
+        LOGGER.info("Checking if messages with headers are correctly mirrored");
+        ClientUtils.waitTillClientFinish(targetConsumerName, NAMESPACE, MESSAGE_COUNT);
+
+        LOGGER.info("Checking log of {} job if the headers are correct", targetConsumerName);
+        String headerFoo = "key: foo, value: bar";
```

Comment:
think the naming of these strings + the header key val pairs is just a tad confusing, I would just call them header1 header2
With key vals header_key_one header_value_one etc...

But just a suggestion.